### PR TITLE
DAOS-8036 test: Add pool properties to TestPool and Dmg

### PR DESCRIPTION
**Pull request by mjean308**
Test-tag-hw-large: pr soak_smoke

Signed-off-by: Maureen Jean <maureen.jean@intel.com>

### CODE REVIEW

**Comment by mjean308** [comment]
Single failure was in soak smoke due to FIO DAOS-7778/7671

File: src/tests/ftest/util/test_utils_pool.py [hunk 1]
```diff
@@ -51,6 +51,7 @@ def __init__(self, context, dmg_command, cb_handler=None):
         self.nvme_size = BasicParameter(None)
         self.prop_name = BasicParameter(None)       # name of property to be set
         self.prop_value = BasicParameter(None)      # value of property
+        self.properties = BasicParameter(None)      # string of cs name:value
```
Comment (shimizukko) [inline src/tests/ftest/util/test_utils_pool.py:54]: What is cs?

**Review by shimizukko — APPROVED** [review]

**Review by dinghwah — APPROVED** [review]
LGTM

File: src/tests/ftest/util/test_utils_pool.py (hunk 1 above)
Comment (mjean308) [reply src/tests/ftest/util/test_utils_pool.py:54]: comma separated name:value pair

**Pull request merged**